src/Whoops/Util/TemplateHelper.php

```diff
@@ -72,6 +72,39 @@ public function escapeButPreserveUris($raw)
         );
     }
 
+    /**
+     * Makes sure that the given string breaks on the delimiter.
+     *
+     * @param  string $delimiter
+     * @param  string $s
+     * @return string
+     */
+    public function breakOnDelimiter($delimiter, $s)
+    {
+        $parts = explode($delimiter, $s);
+        foreach ($parts as &$part) {
```

Comment:
By using a reference php doesn't have to make a copy of the string and you can update the array value without manually updating the array. The following code gives the same result
```php
 foreach ($parts as $k => $part) {
    $parts[$k] = '<div class="delimiter">' . $part . '</div>';
}
```
I used a reference without thinking about it :)